src/main/java/com/playmonumenta/networkchat/ChannelManager.java

```diff
@@ -159,6 +159,10 @@ public static Argument<String> getChannelNameArgument(String argName, ChannelPre
 		return new TextArgument(argName).replaceSuggestions(getChannelNameSuggestions(channelPredicate));
 	}
 
+	public static Argument<String> getChannelDescriptionArgument(String argDescription) {
+		return new TextArgument(argDescription);
```

Comment:
If this is always the last argument, swap it to a `GreedyStringArgument` so that you don't need quotes (and thus don't need to escape quotes inside the text).